tests/unittests/Foundation/NSIndexSetTests.mm

```diff
@@ -27,18 +27,20 @@ void verifyIndexSetEqualsArray(NSArray* expected, NSIndexSet* actual) {
     }];
 
     ASSERT_EQ([expected count], callCount);
-    ASSERT_EQ(YES, [actual isEqualToIndexSet:expectedIndexSet]);
+    ASSERT_TRUE([actual isEqualToIndexSet:expectedIndexSet]);
```

Comment:
Oh well looks like we store std::vector inside the object and that does not pass ASSERT_OBJCEQ check

---
In reply to: [73382204](https://github.com/Microsoft/WinObjC/pull/796#discussion_r73382204) [](ancestors = 73382204)